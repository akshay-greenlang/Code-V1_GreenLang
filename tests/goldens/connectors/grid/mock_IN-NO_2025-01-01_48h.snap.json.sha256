41c28f6e7821bb957da3fbff2262a5c84315d32ec45ba480337c90f1ffde74b8