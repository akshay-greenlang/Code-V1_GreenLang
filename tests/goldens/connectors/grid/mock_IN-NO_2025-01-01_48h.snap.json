{"connector_id":"grid/intensity/mock","connector_version":"0.1.0","payload":{"metadata":{"algorithm":"seasonal_sinusoid","connector":"grid/intensity/mock","data_points":48,"region_offset":50,"seed":"46467e61912467f4366866172e94f25a5224d686f4fd3abe17a3ad97abe7b7ea","version":"0.1.0"},"region":"IN-NO","resolution":"hour","series":[{"quality":"simulated","ts":"2025-01-01T00:00:00Z","unit":"gCO2/kWh","value":"350"},{"quality":"simulated","ts":"2025-01-01T01:00:00Z","unit":"gCO2/kWh","value":"401.8"},{"quality":"simulated","ts":"2025-01-01T02:00:00Z","unit":"gCO2/kWh","value":"450"},{"quality":"simulated","ts":"2025-01-01T03:00:00Z","unit":"gCO2/kWh","value":"491.4"},{"quality":"simulated","ts":"2025-01-01T04:00:00Z","unit":"gCO2/kWh","value":"523.2"},{"quality":"simulated","ts":"2025-01-01T05:00:00Z","unit":"gCO2/kWh","value":"543.2"},{"quality":"simulated","ts":"2025-01-01T06:00:00Z","unit":"gCO2/kWh","value":"550"},{"quality":"simulated","ts":"2025-01-01T07:00:00Z","unit":"gCO2/kWh","value":"543.2"},{"quality":"simulated","ts":"2025-01-01T08:00:00Z","unit":"gCO2/kWh","value":"523.2"},{"quality":"simulated","ts":"2025-01-01T09:00:00Z","unit":"gCO2/kWh","value":"491.4"},{"quality":"simulated","ts":"2025-01-01T10:00:00Z","unit":"gCO2/kWh","value":"450"},{"quality":"simulated","ts":"2025-01-01T11:00:00Z","unit":"gCO2/kWh","value":"401.8"},{"quality":"simulated","ts":"2025-01-01T12:00:00Z","unit":"gCO2/kWh","value":"350"},{"quality":"simulated","ts":"2025-01-01T13:00:00Z","unit":"gCO2/kWh","value":"298.2"},{"quality":"simulated","ts":"2025-01-01T14:00:00Z","unit":"gCO2/kWh","value":"250"},{"quality":"simulated","ts":"2025-01-01T15:00:00Z","unit":"gCO2/kWh","value":"208.6"},{"quality":"simulated","ts":"2025-01-01T16:00:00Z","unit":"gCO2/kWh","value":"176.8"},{"quality":"simulated","ts":"2025-01-01T17:00:00Z","unit":"gCO2/kWh","value":"156.8"},{"quality":"simulated","ts":"2025-01-01T18:00:00Z","unit":"gCO2/kWh","value":"150"},{"quality":"simulated","ts":"2025-01-01T19:00:00Z","unit":"gCO2/kWh","value":"156.8"},{"quality":"simulated","ts":"2025-01-01T20:00:00Z","unit":"gCO2/kWh","value":"176.8"},{"quality":"simulated","ts":"2025-01-01T21:00:00Z","unit":"gCO2/kWh","value":"208.6"},{"quality":"simulated","ts":"2025-01-01T22:00:00Z","unit":"gCO2/kWh","value":"250"},{"quality":"simulated","ts":"2025-01-01T23:00:00Z","unit":"gCO2/kWh","value":"298.2"},{"quality":"simulated","ts":"2025-01-02T00:00:00Z","unit":"gCO2/kWh","value":"350"},{"quality":"simulated","ts":"2025-01-02T01:00:00Z","unit":"gCO2/kWh","value":"401.8"},{"quality":"simulated","ts":"2025-01-02T02:00:00Z","unit":"gCO2/kWh","value":"450"},{"quality":"simulated","ts":"2025-01-02T03:00:00Z","unit":"gCO2/kWh","value":"491.4"},{"quality":"simulated","ts":"2025-01-02T04:00:00Z","unit":"gCO2/kWh","value":"523.2"},{"quality":"simulated","ts":"2025-01-02T05:00:00Z","unit":"gCO2/kWh","value":"543.2"},{"quality":"simulated","ts":"2025-01-02T06:00:00Z","unit":"gCO2/kWh","value":"550"},{"quality":"simulated","ts":"2025-01-02T07:00:00Z","unit":"gCO2/kWh","value":"543.2"},{"quality":"simulated","ts":"2025-01-02T08:00:00Z","unit":"gCO2/kWh","value":"523.2"},{"quality":"simulated","ts":"2025-01-02T09:00:00Z","unit":"gCO2/kWh","value":"491.4"},{"quality":"simulated","ts":"2025-01-02T10:00:00Z","unit":"gCO2/kWh","value":"450"},{"quality":"simulated","ts":"2025-01-02T11:00:00Z","unit":"gCO2/kWh","value":"401.8"},{"quality":"simulated","ts":"2025-01-02T12:00:00Z","unit":"gCO2/kWh","value":"350"},{"quality":"simulated","ts":"2025-01-02T13:00:00Z","unit":"gCO2/kWh","value":"298.2"},{"quality":"simulated","ts":"2025-01-02T14:00:00Z","unit":"gCO2/kWh","value":"250"},{"quality":"simulated","ts":"2025-01-02T15:00:00Z","unit":"gCO2/kWh","value":"208.6"},{"quality":"simulated","ts":"2025-01-02T16:00:00Z","unit":"gCO2/kWh","value":"176.8"},{"quality":"simulated","ts":"2025-01-02T17:00:00Z","unit":"gCO2/kWh","value":"156.8"},{"quality":"simulated","ts":"2025-01-02T18:00:00Z","unit":"gCO2/kWh","value":"150"},{"quality":"simulated","ts":"2025-01-02T19:00:00Z","unit":"gCO2/kWh","value":"156.8"},{"quality":"simulated","ts":"2025-01-02T20:00:00Z","unit":"gCO2/kWh","value":"176.8"},{"quality":"simulated","ts":"2025-01-02T21:00:00Z","unit":"gCO2/kWh","value":"208.6"},{"quality":"simulated","ts":"2025-01-02T22:00:00Z","unit":"gCO2/kWh","value":"250"},{"quality":"simulated","ts":"2025-01-02T23:00:00Z","unit":"gCO2/kWh","value":"298.2"}],"unit":"gCO2/kWh"},"provenance":{"connector_id":"grid/intensity/mock","connector_version":"0.1.0","metadata":{"data_points":48,"end":"2025-01-03T00:00:00+00:00","region":"IN-NO","resolution":"hour","start":"2025-01-01T00:00:00+00:00"},"mode":"record","query_hash":"8860c36bc75cb4609d5317f34177d3e413fd5f18f7657b9cace433dbec574a14","schema_hash":"2cf663a8a03d50acb0c1df2fba40940ffb4f63f0d1882a6386a70320e2bc06f6","seed":"46467e61912467f4366866172e94f25a5224d686f4fd3abe17a3ad97abe7b7ea","snapshot_id":"64c11b1896b5f198e24d08abe63f1605a01009405465aac5733e131dafaa0c7a","timestamp":"2025-10-09T05:13:40.748527Z"},"schema":{"payload_model":"GridIntensityPayload","provenance_model":"ConnectorProvenance"}}